lib/hubspot-api-client.rb

```diff
@@ -1,70 +1,3 @@
-require 'uri'
+require_relative 'hubspot/client'
 require 'logger'
-require 'require_all'
-require_rel 'hubspot/*.rb'
-require_rel 'hubspot/helpers'
-require_rel 'hubspot/discovery'
-require_rel 'hubspot'
-
-module Hubspot
-  class << self
-    CLIENTS = [
-      'Automation::Actions',
-      'Cms::AuditLogs',
-      'Cms::Blogs::Authors',
-      'Cms::Blogs::BlogPosts',
-      'Cms::Blogs::Tags',
-      'Cms::Domains',
-      'Cms::Hubdb',
-      'Cms::Performance',
-      'Cms::SiteSearch',
-      'Cms::SourceCode',
-      'Cms::UrlRedirects',
-      'CommunicationPreferences',
-      'Conversations::VisitorIdentification',
-      'Crm::Associations',
-      'Crm::Companies',
-      'Crm::Contacts',
-      'Crm::Deals',
-      'Crm::Extensions::Accounting',
-      'Crm::Extensions::Calling',
-      'Crm::Extensions::Cards',
-      'Crm::Extensions::Videoconferencing',
-      'Crm::Imports',
-      'Crm::LineItems',
-      'Crm::Objects',
-      'Crm::Objects::Calls',
-      'Crm::Objects::Emails',
-      'Crm::Objects::FeedbackSubmissions',
-      'Crm::Objects::Meetings',
-      'Crm::Objects::Notes',
-      'Crm::Objects::Tasks',
-      'Crm::Owners',
-      'Crm::Pipelines',
-      'Crm::Products',
-      'Crm::Properties',
-      'Crm::Quotes',
-      'Crm::Schemas',
-      'Crm::Tickets',
-      'Crm::Timeline',
-      'Events',
-      'Files::Files',
-      'Marketing::Events',
-      'Marketing::Forms',
-      'Marketing::Transactional',
-      'OAuth',
-      'Settings::Users',
-      'Webhooks',
-    ].freeze
-
-    def configure
-      return Configuration.default unless block_given?
-      yield(Configuration.default)
-      CLIENTS.each do |client_class_name|
-        config_class = Hubspot.const_get("#{client_class_name}::Configuration")
-        yield(config_class.default)
-      end
-      Configuration.default
-    end
```

Comment:
Removing this method means we now don't have a way to centraly configure the client. This is really bad.

Code like this stopped working:
```ruby
Hubspot.configure do |config|
  config.timeout = 2
end
```

Can we get this method back, @plaurynovich-hubspot? It's gone since v15 and still missing in v18.